react-native/react/local-debug.desktop.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+/*
+ * File to stash local debug changes to. Never check this in with changes
+ * @flow
+ */
+
+import { createRouterState } from './reducers/router'
+import * as Tabs from './constants/tabs'
+
+let config = {
+  overrideRouterState: null,
+  overrideActiveTab: null,
+  skipLoginRouteToRoot: false,
+  allowStartupFailure: false
+}
+
+/* eslint-disable no-undef */
+if (process.env.NODE_ENV === 'development' && false) {
```

Comment:
Can you put this in: https://github.com/keybase/client/blob/master/react-native/react/constants/platform.desktop.js and use that?